aiopg/__init__.py

```diff
@@ -11,7 +11,7 @@
            'version', 'version_info', 'DEFAULT_TIMEOUT', 'IsolationLevel',
            'Transaction')
 
-__version__ = '0.15.0'
+__version__ = '0.16.0'
```

Comment:
no need to lift the version in a separate pull request

Reply:
did you mean: "no need to life the version`, it will be done ` in a separate pull request"?

Reply:
Yes, this is usually done release engineer, there may already be several tasks in the issue.

Reply:
ok, reverted